src/components/pages/python.tsx

```diff
@@ -58,10 +81,9 @@ function Python(): JSX.Element {
         </div>
         <div id='output'>
           <div id='dropdowns'>
-            {/* TODO: replace change with actual functionality */}
-            <BooleanDropdown onChange={() => void 0} position={Position.Top}/>
-            <BooleanDropdown onChange={() => void 0} position={Position.Middle}/>
-            <BooleanDropdown onChange={() => void 0} position={Position.Bottom}/>
+            {dropdowns.map((dropdown) => (
```

Comment:
im not sure if this works, but would
```
Object.keys(dropValues).map((dropdown) => {
...
```
work? so we dont need the other `dropdowns` array